.solhint.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "solhint:recommended",
   "rules": {
-    "compiler-version": ["error","^0.6.2"],
+    "compiler-version": ["error",">=0.7.5"],
+    "func-visibility": ["off",{"ignoreConstructors":true}],
```

Comment:
added because solhint told me so...
``` Explicitly mark visibility in function (Set ignoreConstructors to true if using solidity >=0.7.0) [Warning/func-visibility]
```